api/libproject/devstart.rb

```diff
@@ -67,7 +67,7 @@ def make_gae_vars(min_idle_instances = 0, max_instances = 10, instance_class = '
     :config_json => "config_staging.json",
     :cdr_config_json => "cdr_config_staging.json",
     :featured_workspaces_json => "featured_workspaces_staging.json",
-    :gae_vars => make_gae_vars,
+    :gae_vars => make_gae_vars(0, 10, 'F4'),
```

Comment:
This is somewhat expensive - the reason it's enabled in test is because the reporting cron was hitting OOM issues without it - we would like to revert that ASAP. Unless we're seeing the same issue here, I would use F1 (the default).

I suggest also using a lower instance count here, maybe 4 or 5. Test is at least getting constantly hit by multiple concurrent tests - while staging is usually only really used once a week by the releaser.

Reply:
Sounds good. it's a minor change but will lead to improvement.